flatten_join_nested_file.py

```diff
@@ -427,6 +429,21 @@ def drop_foreign_keys(tbl_df):
     tbl_df_drop_sk = tbl_df.drop(*cols)
     return tbl_df_drop_sk
 
+# pivot tags to columns
+def clean_tags(tbl_df):
+    columns_without_tags = tbl_df.columns
+    columns_without_tags.remove("key")
+    columns_without_tags.remove("value")
+    pivot_df = tbl_df.filter((col("key") == "PRODUCT") | \
+                        (col("key") == "COMPONENT") | \
+                        (col("key") == "ASSET") | \
+                        (col("key") == "SERVICE") | \
+                        (col("key") == "ROLE") | \
+                        (col("key") == "ORGANIZATION_ID") | \
+                        (col("key") == "SERVICE_SHORT")).groupby(columns_without_tags).pivot("key").agg(first("value")).dropDuplicates()
```

Comment:
why are those upper case?

Reply:
The Tags keys are upper case in the data.